uploader.py

```diff
@@ -24,7 +24,7 @@
 import urllib
 import urllib2
 from xml.sax.saxutils import quoteattr
-from internetarchive import get_item, upload
+from internetarchive import get_item
```

Comment:
It wasn't used. We're using item.upload() not the actual upload() I think (in fact, we're overwriting this with our own upload function). You can test it to verify this.